cmd/clusternet-scheduler/app/app.go

```diff
@@ -80,3 +93,11 @@ func NewSchedulerCommand(ctx context.Context) *cobra.Command {
 
 	return cmd
 }
+
+// WithPlugin creates an Option based on plugin name and factory. Please don't remove this function: it is used to register out-of-tree plugins,
+// hence there are no references to it from the kubernetes scheduler code base.
```

Comment:
Should rephrase this.

And we should add a reference link, such as "Copied from https://xxxx"